### ISSUE TITLE
Japan: Mountain Day (8/11) from 2016

### ISSUE THREAD

**Issue by suhara** [novafloss/workalendar#131]
8/11 becomes public holiday in Japan from 2016.

Public holidays in Japan - Wikipedia
https://en.wikipedia.org/wiki/Public_holidays_in_Japan

```
>>> from workalendar.asia import Japan
>>> cal = Japan()
>>> cal.holidays(2016)
[(datetime.date(2016, 1, 1), 'New year'),
 (datetime.date(2016, 1, 11), 'Coming of Age Day'),
 (datetime.date(2016, 2, 11), 'Foundation Day'),
 (datetime.date(2016, 3, 20), 'Vernal Equinox Day'),
 (datetime.date(2016, 4, 29), 'Showa Day'),
 (datetime.date(2016, 5, 3), 'Constitution Memorial Day'),
 (datetime.date(2016, 5, 4), 'Greenery Day'),
 (datetime.date(2016, 5, 5), "Children's Day"),
 (datetime.date(2016, 7, 18), 'Marine Day'),
 (datetime.date(2016, 9, 19), 'Respect-for-the-Aged Day'),
 (datetime.date(2016, 9, 22), 'Autumnal Equinox Day'),
 (datetime.date(2016, 10, 10), 'Health and Sports Day'),
 (datetime.date(2016, 11, 3), 'Culture Day'),
 (datetime.date(2016, 11, 23), 'Labour Thanksgiving Day'),
 (datetime.date(2016, 12, 23), "The Emperor's Birthday")]
```

**Comment by brunobord** [comment]
thank you for the report. will try to add this day as soon as possible.

**Comment by brunobord** [comment]
patch committed and released on the version 0.5.0.

**Comment by suhara** [comment]
Great! Thank you!
